container/test/services/webcomponents.service.spec.ts

```diff
@@ -162,7 +162,31 @@ describe('createClientAPI', () => {
       };
       expect(dispatchEventSpy).toHaveBeenCalledWith(Events.NAVIGATION_REQUEST, expectedPayload);
     });
-  
+
+    it('test linkManager navigateToIntent', () => {

```

Comment:
navigateToIntent: (semanticSlug: string, params = {}, settings = {})
 
 This test is good. Could you add more unit tests for other cases. To try and keep the code coverage of the newly introduced functions optimal.
 
 some potential cases to have in unit tests.
  - navigateToIntent(slug)
  - navigateToIntent(slug, params)
  - navigateToIntent(slug, params, settings) 
  -  navigateToIntent(slug, undefined, settings)  
  ...